server/settings.go

```diff
@@ -31,21 +40,48 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 	if connection.Settings == nil {
 		connection.Settings = &ConnectionSettings{}
 	}
-	connection.Settings.Notifications = value
+	switch args[1] {
+	case subCommandAssignee:
```

Comment:
WDYT about having a:

```go
connection.Settings.SendNotificationsFor(role, &value)
```

Then we wouldn't need these long switches and I think it would be easier to maintain? Let me know your thoughts!

Reply:
The problem is not the switch case itself in my opinion, but the changes in `ConnectionSettings` to support the notifications, let me show you what I mean tomorrow. I think we could try to make it easier to handle.